### DISCUSSION TITLE
How to add Aptos's SDK into Cocos Creator ?

### DISCUSSION THREAD

**Discussion by ChinhQT** [aptos-labs/aptos-developer-discussions#158] [Questions]
I want to add the Aptos's SDK  into Cocos Creator because I want to try to use this Aptos's Wallet. Thank you very much.

**Comment by Alexzander-Stone** [comment]
Are you asking how to integrate/call the SDK/API from within Cocos?
If so, you'll either use the pre-existing [TS SDK](https://github.com/aptos-labs/aptos-ts-sdk), or you'll need to interact directly with the API's (can base your logic flow on our SDKs, nonetheless). Since Cocos supports Typescript, the former should be possible. 

In the case that you're not asking the above question, please explain further with more context.

**Comment by ChinhQT** [comment]
<script src="https://unpkg.com/@aptos-labs/ts-sdk/dist/browser/index.global.js" />
Cannot find "/dist/browser/index.global.js" in @aptos-labs/ts-sdk@1.13.1
https://discuss.cocos2d-x.org/t/how-to-send-usdt-token-using-metamask-in-cocos-creator-3-7-3/59366

Currently, I still resolve the problems. I need to know the root cause to resolve.

**Reply by Alexzander-Stone** [reply]
Hi @ChinhQT, the `src` attribute is using a path that isn't available within our package.

You can check out the available contents, [here](https://www.npmjs.com/package/@aptos-labs/ts-sdk?activeTab=code).
